docs/toc.js

```diff
@@ -263,6 +263,9 @@ module.exports = {
         ['Using Unicode with Appium', 'unicode.md'],
         ['Troubleshooting', 'troubleshooting.md']
       ]],
+      ['Tutorial', ['tutorial',
```

Comment:
Could you add other brilliant documentation before emerging this?

Reply:
@KazuCocoa what you mean? write something more? e.g. how about working with input fields?

Reply:
I mean adding documentation like tutorial/swipe/swipe-troubleshoot-guide.md as below.

```
['Tutorial', ['tutorial',
        ['swift', ['switt',
            ['swipe trouble shoot', 'wipe-troubleshoot-guide.md']
         ]]
...       
```

They will be part of https://appium.io/docs/en/writing-running-appium/caps by https://github.com/appium/appium.io and pushed to appium.io

Reply:
@KazuCocoa pls check

Reply:
Awesome. It looks good to me. (and this CI also passed)
Will arrange links if we need some tweaks later.